devon.json

```diff
@@ -0,0 +1,28 @@
+{
+	"response": [
+		{
+			"maintainer": "Jay (DhruvOz)",
+			"oem": "Motorola",
+			"device": "Motorola G32",
+			"filename": "crDroidAndroid-15.0-20241119-devon-v11.0-BETA.zip",
+			"download": "https://sourceforge.net/projects/crdroid/files/devon/11.x/crDroidAndroid-15.0-20241119-devon-v11.0-BETA.zip/download",
+			"timestamp": 1732033516,
+			"md5": "9b0e2070e8d7e2deeb597cf584b52899",
+			"sha256": "1ed399576621065812f41d65d52a54593d2534acce00cb6acbbdae9454548304",
+			"size": 1113794269,
+			"version": "11.0",
+			"buildtype": "Monthly",
+			"forum": "https://xdaforums.com/t/rom-14-0-official-moto-g32-devon-crdroid-v10-0-08-01-2024.4649906/",
```

Comment:
Update forum template as per new template
Features points to 12.0
Also follow back to our donation page please (yours can stay also, just add ours also)